test/e2e/gardener/seed/renew_gardenlet_kubeconfig.go

```diff
@@ -30,10 +30,18 @@ var _ = Describe("Seed Tests", Label("Seed", "default"), func() {
 		var seed *gardencorev1beta1.Seed
 
 		BeforeEach(func() {
-			// find the first seed, it doesn't matter which one (seed name differs between test scenarios, e.g., non-ha/ha)
+			// Find the first seed which is not "e2e-managedseed". Seed name differs between test scenarios, e.g., non-ha/ha.
+			// However, this test should not use "e2e-managedseed", because it is created and deleted in a separate e2e test.
+			// This e2e test already includes tests for the "Renew gardenlet kubeconfig" functionality. Additionally,
+			// it might be already gone before the kubeconfig was renewed.
 			seedList := &gardencorev1beta1.SeedList{}
-			Expect(testClient.List(ctx, seedList, client.Limit(1))).To(Succeed())
-			seed = seedList.Items[0].DeepCopy()
+			Expect(testClient.List(ctx, seedList)).To(Succeed())
+			for _, s := range seedList.Items {
+				if s.Name != "e2e-managedseed" {
```

Comment:
Should we define a constant and use it in https://github.com/gardener/gardener/blob/768f682fd4346d5fc2b36342747422c5c6fe5e0a/test/e2e/gardener/managedseed/create_rotate_delete.go#L56 
as well?

Reply:
good idea 😄 
While changing that, I found the same issue in `Garden Cluster Access For Seed Components` e2e test and applied the logic here as well